volatility/framework/symbols/windows/extensions/__init__.py

```diff
@@ -692,11 +692,17 @@ def is_valid(self) -> bool:
             if self.UniqueProcessId % 4 != 0:
                 return False
 
-            if self.Pcb.DirectoryTableBase == 0:
+            # check for all 0s besides the PCID entries
+            if isinstance(self.Pcb.DirectoryTableBase, objects.Array):
+                dtb = self.Pcb.DirectoryTableBase.cast("unsigned long long")
```

Comment:
Looks good, I wonder whether just always casting it to the right value is worthwhile, but I think this way is better for two reasons:
a) technically `unsigned long long` may not be defined by the windows symbol table (strange, but can happen)
b) always casting something will have a (extremely small) performance impact, so probably better to the isinstance to make sure first.

I can't figure out if it's better to unify this and the other existing code, but if so where would it live?  5:S  Hmmm, probably fine for now, but if we need to do it a third time, we should definitely put it in a single place...  5:)